pages/api/create-offer.ts

```diff
@@ -0,0 +1,166 @@
+import Cookies from "cookies";
+import type { NextApiRequest, NextApiResponse } from "next";
+import {
+	FormError,
+	OfferParameters,
+	publicQuery,
+	PublicQueryResult,
+} from "../../lib/shared";
+import { validateOffer } from "../../lib/validateOffer";
+import generateUniqueCode from "../../lib/generateUniqueCode";
+
+const fetchTypes = async (): Promise<PublicQueryResult> => {
+	const response = await fetch(process.env.NEXT_PUBLIC_CONTEMBER_CONTENT_URL!, {
```

Comment:
add try catch (on any level but somwhere)

Reply:
you mean for example wrap the fetch call inside try{}? what should i put inside catch block? there are no try..catch in another api/* files so I don't get the idea

Reply:
`response.json()` can be undefined when `fetch` fails
but if it's consistent with rest of the code - no need to solve it better way